flink-runtime/src/main/java/org/apache/flink/runtime/jobmaster/DefaultJobMasterServiceProcess.java

```diff
@@ -214,15 +214,6 @@ public void jobReachedGloballyTerminalState(ExecutionGraphInfo executionGraphInf
         resultFuture.complete(JobManagerRunnerResult.forSuccess(executionGraphInfo));
     }
 
-    @Override
```

Comment:
Attention:  I didn't see any product code use this method, so i delete it in this pr. Please help make sure for safe. @tillrohrmann

Reply:
I also couldn't find another user of this method. Hence, I believe it is ok to remove it.